go.mod

```
@@ -4,8 +4,8 @@ go 1.20
 
 require (
 	github.com/IOTechSystems/onvif v0.1.6
-	github.com/edgexfoundry/device-sdk-go/v3 v3.0.0-dev.54
-	github.com/edgexfoundry/go-mod-bootstrap/v3 v3.0.0-dev.47
+	github.com/edgexfoundry/device-sdk-go/v3 v3.0.0-dev.57
```

This will allow the removal of TOML module from the Attribution.txt file
```suggestion
	github.com/edgexfoundry/device-sdk-go/v3 v3.0.0-dev.60
```

---

> This will allow the removal of TOML module from the Attribution.txt file

I don't think https://github.com/edgexfoundry/device-usb-camera/pull/203/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R6 helped to achieve the above?

---

there is error when updating to dev.60, WIP